index.html

```diff
@@ -208,7 +208,7 @@ <h3>Skills</h3>
     <footer>
```

Comment:
- [OPTIONAL] I highly suggest removing the attribute value from the form inputs because in case the user only added a text message the form will submit without asking for the user's full name and email.

![image](https://user-images.githubusercontent.com/68030297/221371515-77bc5c3f-4306-4fdb-8f14-8888b764a764.png)

Those are default values and the form will be submitted without the user information, and from the project requirement you just need to apply a placeholder only 👍🏼